### PR TITLE
Ban specifications without an audience.

### PR DESCRIPTION
**Pull request by jyasskin**
With reference to the W3C Process's requirement for implementation experience.

I think this satisfies Maciej's request. What do you think?

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -377,6 +377,13 @@ <h2 id=process>Process</h2>
 <a href=https://www.w3.org/community/about/agreements/cla/>W3C Community
 Contributor License Agreement (CLA)</a>.
 
+<p>The Community Group will not work on or publish a <a
```
Comment (cwilso) [inline charter.html]: I'd rather see this just say "publish", not "work on or publish".

File: charter.html [hunk 2]
```diff
@@ -377,6 +377,13 @@ <h2 id=process>Process</h2>
...
+href="https://www.w3.org/community/reports/">specification</a> unless the
+maintainers of at least two independent interoperable implementations have
```
Comment (cwilso) [inline charter.html]: how about "unless at least two organizations have volunteered to provide implementation experience"?

**Review by cwilso — CHANGES_REQUESTED** [review]

File: charter.html (hunk 1 above)
Comment (jyasskin) [reply charter.html]: Works for me, done.

**Review by jyasskin — COMMENTED** [review]
Both done.

**Comment by cwilso** [comment]
LGTM, thanks @jyasskin!

**Pull request merged**